dev-db/sqlitebrowser/sqlitebrowser-3.8.0-r1.ebuild

```diff
@@ -21,7 +21,7 @@ RDEPEND="
 	dev-cpp/antlr-cpp:2
 	dev-db/sqlite:3
 	dev-libs/qcustomplot[qt5=]
-	x11-libs/qscintilla
+	x11-libs/qscintilla:=
```

Comment:
In theory yes, the problem is, that ebuild is stable and can depend on `qt4` and `qt5`. If I dispatch based on qt4 and qt5, then `x11-libs/qscintilla:=[qt5(+)]` will break the tree, as there is no stable qscintilla with Qt5 yet. My suggestion: Lets merge this, stable my qt4/qt5 qscintilla, and then properly fix that ebuild.